packages/mdc-grid-list/package.json

```diff
@@ -4,6 +4,9 @@
   "description": "The Material Components for the web grid list component",
   "license": "MIT",
   "main": "dist/mdc.gridList.js",
+  "module": "index.js",
+  "sideEffects": false,
+  "types": "dist/gridList.d.ts",
```

Comment:
From this point on in the PR it seems like the `types` entries are missing the `mdc.` prefix to be consistent with `main`